tools/setup_env.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# Script setups the environment for the Ziggurat test suite
```

Comment:
```suggestion
# This script sets up the environment for the Ziggurat test suite.
```

Reply:
also, is this based on some Algorand resource? if so, we should ink it

Reply:
We could link this [page](https://developer.algorand.org/docs/clis/goal/network/create/?from_query=goal%20network%20cr#create-publication-overlay) maybe here:

It describes how to create a network with a goal command. And we are using that command in the script.

Reply:
Done